setup.py

```diff
@@ -97,4 +97,4 @@
           'Topic :: Software Development :: Code Generators',
           'Topic :: Scientific/Engineering',
           'Topic :: Utilities'],
-      install_requires=['mako', 'pyxb', 'pytest', 'scipy'])
\ No newline at end of file
+      install_requires=['mako', 'pyxb>=1.2.5', 'pytest', 'scipy'])
```

Comment:
Hi @thorade ,

thanks for that pull request. However, I won't accept it ;-).

pyxb provides no backwards compatibility. That means, a binding (e.g. CityGML or TypeElements in TEASER) generated with pyxb version 1.2.5 won't work with pyxb version 1.2.6. If there is a new release of pyxb ,a new installation of TEASER wouldn't work without updating the bindings. To conclude: we actually always need a distinct verison of pyxb, So changing your code to `pyxb==1.2.5` would add value.